presidio-anonymizer/presidio_anonymizer/entities/anonymizer_request.py

```diff
@@ -18,32 +18,32 @@ class AnonymizerRequest:
     def __init__(self, data: dict, anonymizers):
         """Handle and validate data for the text replacement.
 
-        :param data: a map which contains the transformations, analyzer_results and text
+        :param data: a map which contains the anonymizers, analyzer_results and text
         """
         self.anonymizers = anonymizers
-        self._transformations = {}
+        self._anonymizers = {}
         self._analysis_results = AnalyzerResults()
         self.__validate_and_insert_input(data)
-        self.default_transformation = {
+        self.default_anonymizer = {
             "type": "replace",
             "anonymizer": self.anonymizers["replace"],
         }
 
-    def get_transformation(self, entity_type: str):
+    def get_anonymizer_dto(self, entity_type: str):
         """
-        Get the right transformation from the list.
+        Get the right anonymizer_dto from the list.
```

Comment:
Why dto? Isn't it a class which holds the logic for anonymization?

Reply:
This is a prep for an additonal work Shiran will do to change the anonymizer engine to receive parameters instead of dict.